MetaMask/metamask-extension
# feat: :sparkles: show a notification item in the settings page

## **Description**

This PR introduces the `Notifications` option within the `settings page`. Clicking on this option will render the user to the `notifications/settings` page.
Specifically:
1. If basic functionalities are not active, the option disappears from the menu
2. If notifications are not active, the user will still be redirected to the `notifications/settings page`, where they can enable them
3. If notifications are active, the user will access the `notifications/settings` page, where they can select their options

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26843?quickstart=1)

## **Related issues**

N/A

## **Manual testing steps**

1. Go to the settings page.
2. Find the `Notifications` option
3. Verify that the `Notifications` option links to the `notifications/settings` page
4. Verify that the `Notifications` option is hidden if basic functionalities are off

## **Screenshots/Recordings**

### **Before**

<img width="1624" alt="Screenshot 2024-10-23 at 16 26 12" src="https://github.com/user-attachments/assets/c260ca28-536d-413b-862b-79deda5e70bb">

### **After**

<img width="1624" alt="Screenshot 2024-10-23 at 16 41 30" src="https://github.com/user-attachments/assets/99fb68d6-ace1-4ddd-a863-7b4e2e125074">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
